go/mysql/mysql_throttle_metric.go

```diff
@@ -11,9 +11,37 @@ import (
 	"time"
 
 	"github.com/outbrain/golib/sqlutils"
+	"github.com/patrickmn/go-cache"
 	metrics "github.com/rcrowley/go-metrics"
 )
 
+var mysqlMetricCache = cache.New(cache.NoExpiration, 10*time.Millisecond)
+
+func getMySQLMetricCacheKey(probe *Probe) string {
+	return fmt.Sprintf("%s:%s", probe.Key, probe.MetricQuery)
```

Comment:
when there is no 'MetricQuery', the cache key would be "probe:", I guess that's ok? should we default to a key instead?
Not big deal though.

Reply:
That's fine. We happen to always a have a `MetricQuery`, and the default at this time is to run `SHOW SLAVE STATUS` ; so the default `''` is in itself a unique identification of a particular query.